api/turing/service/mlp_service.go

```diff
@@ -48,7 +48,11 @@ type merlinClient struct {
 
 func newMerlinClient(googleClient *http.Client, basePath string) *merlinClient {
 	cfg := merlin.NewConfiguration()
-	cfg.BasePath = basePath
+	cfg.Servers = merlin.ServerConfigurations{
+		{
+			URL: basePath,
+		},
+	}
```

Comment:
These changes reflect the changes to the Merlin API server client (autogenerated using the OpenAPI generator) in the newer versions of Merlin.